css/style.css

```diff
@@ -234,7 +234,7 @@ li {
   text-decoration: none;
 }
 
```

Comment:
- [ ] I kindly suggest that you kindly make use of responsive widths instead of making use `overflow: hidden` to hide the horizontal overflow on your web page. See my video below for more explanation:

https://www.loom.com/share/71b29bd9c108446db9d0ed53ea744514

https://github.com/joramot/capstone_portfolio/blob/0741e1838c320bea0cfcc437a8e31ef63d9fc271/css/style.css#L6-L8